### ISSUE TITLE
Deploy to Maven Central

### ISSUE THREAD

**Issue by vaadin-miki** [vaadin-miki/super-fields#366]
The time has come to deploy to Central, as Vaadin Directory seems to behave a bit funky.

Investigate how to automate deployment process so that it deploys to Maven Central.

Note: this will require renaming the source packages, as `org.vaadin.miki` was merely a convention.

**Comment by jonasrotilli** [comment]
I tried to use it and I couldn't.
In the vaadin directory appears:
"This component or version has not been published in Maven."

And when I try to do as it is here in the documentation:
```
<dependency>
      <groupId>org.vaadin.miki</groupId>
      <artifactId>superfields</artifactId>
      <version>0.13.0</version>
 </dependency>
```

In the pom.xml file, it doesn't recognize it.
I don't understand much, but there is something Maven-related to do.

**Comment by vaadin-miki** [comment]
@jonasrotilli there is something fishy with the directory, as I cannot release version 0.13 there and 0.12 seems to be unvailable as well; I informed the Vaadin theam and am waiting for more information

in the mean time, starting the procedures to have a group id in central

**Comment by vaadin-miki** [comment]
@jonasrotilli issue with Vaadin directory resolved, both `0.13.0` and `0.13.0-vaadin14` are now available
version `0.14.0` though will also be in central
